src/main/java/seedu/duke/data/timetable/Event.java

```diff
@@ -217,6 +219,96 @@ public String toString() {
                 + lineSeparator + repeatingString;
     }
 
+    /**
+     * Provides an ArrayList representation of the event for the formatter to use.
+     *
+     * @return ArrayList of strings to represent the event.
+     */
+    public ArrayList<String> toStringArray() {
+        ArrayList<String> result = new ArrayList<>();
+        result.add("Event: " + title);
+        result.add("Date: " + date.toString() + "\tTime: " + time.toString());
+        result.add("Reminder: " + isToRemind);
+        result.add("Repeating: " + isRecurring);
+        return result;
+    }
+
+    /**
+     * Converts an array list of events to an indexed list of events. Each line of event is converted to use
+     * arraylist for the formatter.
+     *
+     * @param events Events to be listed.
+     * @return ArrayList String representation of the list of events.
+     */
+    public static ArrayList<String> calendarToString(ArrayList<Event> events) {
+        ArrayList<String> results = new ArrayList<>();
+        events.sort(Comparator.comparing(Event::getDate).thenComparing(Event::getTime));
+        int i = 1;
+        for (Event event : events) {
+            ArrayList<String> eventString = event.toStringArray();
+            eventString.set(0, i++ + ". " + eventString.get(0));
+            results.addAll(eventString);
+            results.add(" ");
+        }
+        results.remove(results.size() - 1);
+        return results;
+    }
```

Comment:
This should be formatEvents or formatTimetable. See suggestion for formatEvent(event) in the #108